### ISSUE TITLE
Client application fails when no steam directory is present

### ISSUE THREAD

**Issue by ajgon** [JMBeresford/retrom#208]
### Describe the bug
I'm running retrom client 0.4.9 from AppImage, and it fails if no steam directory is present. However if I create dummy one (even empty), it still throws exceptions, but this time, they're handled, app process further and everything works:

```bash
$ ./Retrom_0.4.9_amd64.AppImage
thread 'main' panicked at packages/client/src/main.rs:90:10:
error while running tauri application: PluginInitialization("steam", "Failed validating steam dir. Error: The Steam installation directory either isn't a directory or doesn't exist")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

$ mkdir ~/.local/share/Steam

$ ./Retrom_0.4.9_amd64.AppImage
thread '<unnamed>' panicked at /home/runner/work/retrom/retrom/plugins/retrom-plugin-steam/src/lib.rs:43:22:
Could not get steam library paths: Parse { kind: LibraryFolders, error: ParseError { inner: Missing }, path: "/home/ajgon/.local/share/Steam/steamapps/libraryfolders.vdf" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Could not determine the accessibility bus address
   INFO  Server host: http://localhost:5101
    at plugins/retrom-plugin-service-client/src/desktop.rs:60
    in get_installation_state
```

### Steps To Reproduce
1. Run client without steam installed and steam directories present
2. App fails
3. Create `~/.local/share/Steam` directory
4. Run app again
5. Now everything works

### Expected behavior
Client app should work on setups without Steam installed.

### Environment (please complete the following information _where applicable_):
 - OS: NixOS
 - Browser Firefox
 - Version: 0.4.9

### Logs
Attached in bug description.

**Comment by JMBeresford** [comment]
@ajgon Looks like this was still an issue -- I believe it should be fixed in next release (v0.7.2) in case you are still struggling with needing the workaround

**Comment by ajgon** [comment]
> [@ajgon](https://github.com/ajgon) Looks like this was still an issue -- I believe it should be fixed in next release (v0.7.2) in case you are still struggling with needing the workaround

Turned out, I never removed this directory after reporting the issue, so it worked for me since then 😅 

Bumped to 0.7.2, removed directory and everything works as expected. Thanks! 👍
